src/Compilers/CSharp/Portable/Binder/ImplicitlyTypedLocalBinder.cs

```diff
@@ -1,65 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.CSharp
-{
-    /// <summary>
-    /// This binder is for binding the initializer of an implicitly typed
-    /// local variable. While binding an implicitly typed local variable
-    /// it is illegal to refer to the variable.
-    /// </summary>
-
-    internal sealed class ImplicitlyTypedLocalBinder : Binder
```

Comment:
It will have to be fixed if we ever add a deconstruction declaration to VB. I'll create an issue for it (#13001).

---
In reply to: [73919773](https://github.com/dotnet/roslyn/pull/12991#discussion_r73919773) [](ancestors = 73919773)